lib/modules/txt.mjs

```diff
@@ -456,7 +456,7 @@ export async function validateCopyrightDate (doc, { mode = MODES.NORMAL, year }
 }
 
 /**
- * Validate submission compliance line
+ * Validate submission compliance line page
```

Comment:
Please review this thing in pr that it comes from, Because later it is hard to decompose commits on pull requests for customer review